### ISSUE TITLE
Input doesn't have focus outline in number input in swaps in extension

### ISSUE THREAD

**Issue by georgewrmarshall** [georgewrmarshall/metamask-extension#9]
### Describe the bug
Input doesn't have focus outline in number input in swaps in extension.

### Browser
Chrome

### Operating System
macOS

### Steps to Reproduce
1. Navigate to the MetaMask extension homepage.
2. Go to the "Swaps" section.
3. Select the "Amount" input field.
4. Notice that no focus outline is displayed when interacting with the input.
